docs/example-app/script/index.js

```diff
@@ -130,6 +130,10 @@ define([
     var liveSupport = 'seekable';
     var enableSubtitles = false;
 
+    window.bigscreenPlayer = {
+      playbackStrategy : 'nativehtml5strategy'
```

Comment:
Is this not suppose to be nativestrategy?

Reply:
Reverted it back to MSE strategy (was used for native testing purposes). The intention was to keep the example-app MSE.